### ISSUE TITLE
Grammar on Blog post

### ISSUE THREAD

**Issue by hbutau** [pyung/pycon-ng#1]
**May you please correct the grammar on the sentence in bold:**

**In furtherance of her mission, it has done so** in partnering with other organizations like the Carrington Youth Fellowship Initiative for the Smart Girls project and the Data Science Society of Nigeria for the Data Science Bootcamp.

**Comment by shante665** [comment]
Thanks @hbutau what would you suggest should be the correction to the grammar.

**Comment by takwas** [comment]
I suppose @hbutau refers to the conflicting use of "her" and "it". Correct me if I'm wrong @hbutau.

**Comment by hbutau** [comment]
yes thats right, maybe I got lost in the context. Should it be 'it' or 'her'?

On 23/07/2017, Tosin Damilare James Animashaun <notifications@github.com> wrote:
> I think it @hbutau refers to the conflicting use of "her" and "it". Correct
> me if I'm wrong @hbutau.
>
> --
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/pyung/pycon-ng/issues/1#issuecomment-317229306


-- 
Twitter <http://twitter.com/hamub>
Linkedin <http://www.linkedin.com/in/hamub>
Github <https://github.com/hbutau>
